### PR TITLE
Review mimc

### PR DESCRIPTION
**Pull request by AntoineRondelet**
Modifications related to the review of the pull request #34

### CODE REVIEW

File: src/circuits/commitments/commitments.hpp [hunk 1]
```diff
@@ -10,30 +10,30 @@
...
                 libsnark::pb_variable_array<FieldT> y,
-                std::shared_ptr<libsnark::digest_variable<FieldT>> result,  // sha256(x || y)
+                std::shared_ptr<libsnark::digest_variable<FieldT>> result, // sha256(x || y)
                 const std::string &annotation_prefix = "COMM_gadget");
     void generate_r1cs_constraints();
     void generate_r1cs_witness();
 };
 
-template<typename FieldT>
+template<typename FieldT, typename HashT>
```
Comment (rrtoledo) [inline src/circuits/commitments/commitments.hpp]: HashT not needed.

File: src/circuits/commitments/commitments.hpp [hunk 2]
```diff
@@ -10,30 +10,30 @@
...
 libsnark::pb_variable_array<FieldT> get128bits(libsnark::pb_variable_array<FieldT>& inner_k);
 
 // As mentioned in Zerocash extended paper, page 22
 // Right side of the hash inputs to generate cm is: 0^192 || value_v (64 bits)
-template<typename FieldT>
+template<typename FieldT, typename HashT>
```
Comment (rrtoledo) [inline src/circuits/commitments/commitments.hpp]: HashT not needed.

File: src/circuits/commitments/commitments.hpp (hunk 1 above)
Comment (AntoineRondelet) [reply src/circuits/commitments/commitments.hpp]: Thanks

File: src/circuits/commitments/commitments.hpp (hunk 2 above)
Comment (AntoineRondelet) [reply src/circuits/commitments/commitments.hpp]: Thanks

**Pull request merged**